docs/source/developing-volttron/developing-agents/using-asyncio-in-agents.rst

```diff
@@ -41,8 +28,8 @@ coroutine is shown below:
         # do things
 
 
-The OpenADRVenAgent has a VEN client and this coroutine is added to that client. To create the Event Loop, the OpenADRVenAgent has a helper method
-called '_start_asyncio_loop', which will be used to create tasks from the aforementioned coroutine and run the Event Loop. An example of this method is shown below:
+To create the Event Loop, the OpenADRVenAgent has a helper method called '_start_asyncio_loop', which will be used to
+create tasks from the aforementioned coroutine and run the Event Loop. An example of this method is shown below:
```

Comment:
you use the word tasks....what is a task in this context? a function call?

Reply:
Task is the return object of the function call: `loop.create_task(self.ven_client.run()`. I'm going to remove that verbiage and replace it with something more clear.